### PR TITLE
Read

### PR DESCRIPTION
**Pull request by Feven90**

### CODE REVIEW

**Comment by rousell** [comment]
So, the first thing that I see is that in your App.js you are creating separate PrivateRoute components for each one of your routes. You can actually reuse the PrivateRoute component that you have for all of those. Let me know if you have questions about that.

File: src/helpers/data/weatherRequests.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const getWeather = uid => new Promise((resolve, reject) => {
           weatherObject[weatherId].id = weatherId;
           weatherArray.push(weatherObject[weatherId]);
         });
+        console.log(weatherArray);
```
Comment (rousell) [inline src/helpers/data/weatherRequests.js]: given that this code is going into master, please remove the console.log

**Comment by rousell** [comment]
@Feven90  If you go ahead and fix those things, then this is good to merge! Please come to me if you have questions.

**Comment by Feven90** [comment]
@rousell I deleted the console.log and duplicated private routes

**Comment by rousell** [comment]
@Feven90 This looks good! 👍 merging

**Pull request merged**